spec/setup_acceptance_node.pp

```diff
@@ -34,14 +34,14 @@
   umask     => '0022',
 }
 -> exec { 'Generate CSR':
-  command   => "openssl req -nodes -new -newkey rsa:2048 -subj '/CN=admin' -out '${client_csr}' -keyout '${client_key}'",
+  command   => "openssl req -nodes -new -newkey rsa:2048 -subj '/CN=${facts['networking']['fqdn']}' -addext 'subjectAltName = DNS:${facts['networking']['fqdn']}' -out '${client_csr}' -keyout '${client_key}'",
   path      => ['/bin', '/usr/bin'],
   creates   => $client_csr,
   logoutput => 'on_failure',
   umask     => '0022',
 }
 -> exec { 'Sign CSR':
-  command   => "openssl x509 -req -days 360 -in '${client_csr}' -CA '${ca_cert}' -CAkey '${ca_key}' -CAcreateserial -out '${client_cert}'",
+  command   => "openssl x509 -req -days 360 -in '${client_csr}' -CA '${ca_cert}' -CAkey '${ca_key}' -CAcreateserial -out '${client_cert}' -extfile <(echo 'subjectAltName = DNS:${facts['networking']['fqdn']}')",
```

Comment:
openssl (pre 3.2, which is what CS9 has, and that's what has been passing) doesn't copy extensions from the CSR, so the resulting cert ends up being invalid.

Reply:
and no, I have no idea why the `admin` one worked before…